Ports/dungeonrush/package.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=DungeonRush
+version=1.1-beta
+useconfigure=true
+files="https://github.com/Rapiz1/DungeonRush/archive/refs/tags/v${version}.tar.gz v${version}.tar.gz 295b83cb023bf5d21318992daee125399892bdf16a87c835dfc90b841c929eda"
+auth_type=sha256
+configopts="-DCMAKE_TOOLCHAIN_FILE=$SERENITY_SOURCE_DIR/Toolchain/CMake/CMakeToolchain.txt"
+depends="SDL2 SDL2_image SDL2_mixer SDL2_ttf SDL2_net"
+launcher_name="DungeonRush"
+launcher_category=Games
+launcher_command=/opt/DungeonRush/dungeon_rush
+
+configure() {
+    run cmake $configopts
+}
+
+install() {
+    run mkdir -p "${SERENITY_INSTALL_ROOT}/opt/DungeonRush"
+    run cp -r bin/dungeon_rush res "${SERENITY_INSTALL_ROOT}/opt/DungeonRush"
+    if command -v convert >/dev/null; then
+        run convert dungeonrush.png -resize 32x32 app-32x32.png
+        run convert dungeonrush.png -resize 16x16 app-16x16.png
+        run objcopy --add-section serenity_icon_s="app-16x16.png" "${SERENITY_INSTALL_ROOT}/opt/DungeonRush/dungeon_rush"
+        run objcopy --add-section serenity_icon_m="app-32x32.png" "${SERENITY_INSTALL_ROOT}/opt/DungeonRush/dungeon_rush"
```

Comment:
Adding support for resizing `.png` files to `install_icon` might be trivial?

Reply:
Aye, will fix this in a moment. :)